darts/models/theta.py

```diff
@@ -94,9 +96,12 @@ def fit(self, series: TimeSeries, component_index: Optional[int] = None):
         b_theta = np.polyfit(np.array([i for i in range(0, self.length)]), (1.0 - self.theta) * new_ts.values(), 1)[0]
 
         # Normalization of the coefficient b_theta.
-        self.coef = b_theta / (2.0 - self.theta)
+        self.coef = b_theta / (2.0 - self.theta)  # change to b_theta / (-self.theta) if classical theta
 
         self.alpha = self.model.params["smoothing_level"]
+        if self.alpha == 0.:
+            self.model = hw.SimpleExpSmoothing(new_ts.values()).fit(initial_level=0.2)
```

Comment:
I don't feel too comfortable with this magic number. Could we at the very least put it as a constant at the top of the file?
How did you pick 0.2?

Reply:
It is not a perfect solution, but a temporary one. alpha should be bounded in ]0, 1[, and is now [0,1] in statsmodels.
0.2 is just good practice to reach good results, and gave similar results as the R function.